tractor/_actor.py

```diff
@@ -171,11 +171,6 @@ class Actor:
     _root_nursery: trio.Nursery
     _server_nursery: trio.Nursery
 
-    # marked by the process spawning backend at startup
-    # will be None for the parent most process started manually
-    # by the user (currently called the "arbiter")
-    _spawn_method: Optional[str] = None
```

Comment:
Ahh I understand why you were confused now. Since the `Actor` is normally only instantiated *once* per process it doesn't matter if this is a `class` or instance variable.